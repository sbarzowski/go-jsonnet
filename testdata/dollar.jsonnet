{x: 42, y: {z: $.x}}
