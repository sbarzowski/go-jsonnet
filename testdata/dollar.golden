{
   "x": 42,
   "y": {
      "z": 42
   }
}
